themes/bootstrap3/templates/_ui/components/confirm-menu.phtml

```diff
@@ -0,0 +1,42 @@
+<?php
+  $toggleLabel = $this->transEsc($this->label ?? '{->label}');
+  $toggleClass = trim('confirm__toggle ' . $this->escapeHtmlAttr(($this->toggleClass ?? 'btn')));
+
+  $desc = $this->transEsc($this->description ?? 'Are you sure?');
```

Comment:
`Are you sure?` is not an existing translation string. Can we find something that already exists so we don't need to expand the language files? Since translation is already underway for 9.0, it is not currently possible to add new strings unless we defer this to 9.1.

Reply:
(One interim solution could be to have the template throw an exception for an undefined label for now, and introduce a default in 9.1 -- I did a quick skim of en.ini and didn't see any particularly promising existing strings).